Release: 2.5.6 annoucement (#425)
mixxxdj/website: content/news/XXXX-XX-XX-mixxx-2_5_5-released.md

title: Mixxx 2.5.5 Released
authors: Antoine Colombier
tags: 2.5.5, release announcement
comments: yes
date: 2026-03-06 22:54:46

#### Mixxx 2.5.5 Release Announcement

We're proud to announce a new stable release of Mixxx: version 2.5.5. This version contains updates and fixes for issues, as well as improvements to effects, controller mappings, and overall stability. This should be the last 2.5 release.
We'd like to thank all users for their feedback and emphasize once again the importance of testing and reporting. Please join our [team](https://mixxx.org/get-involved/) to help make Mixxx even better.

Enjoy Mixxx!

#### Important updates and fixes in 2.5.5

- The 'not' operator in the search function was not working correctly.
- Rhythmbox imports were not functioning as expected.

> - A warning about "dataChanged() called with an invalid index range" has been resolved.

Reviewer: ```suggestion
```
(who cares.. ; )

- History playlists now allow track file export.
- Performance when restoring large track selections has been improved.
- White Noise and Echo effects have been enhanced, and crackling in QuickEffect has been fixed.
- Controller mappings for Numark Mixtrack 3, Pioneer CDJ-350, Reloop Beatmix 2/4, and Traktor Kontrol Z1 have been updated.
- Scratching with keylock enabled has been fixed.
- Flatpak packaging files have been added for easier installation on Linux.

> #### 2.5.5 Changelog

Reviewer 2: ```suggestion
#### 2.5.6 Changelog
```

> The complete changelog can be found [here](https://github.com/mixxxdj/mixxx/milestone/53).

Reviewer 2: ```suggestion
The complete changelog can be found [here](https://github.com/mixxxdj/mixxx/blob/2.5.6/CHANGELOG.md).
```
